AirCasting/CreateSessionViews/SelectPeripheralView/SelectPeripheralView.swift

```diff
@@ -69,6 +69,8 @@ struct SelectPeripheralView: View {
             Button(action: {
                 selection = availableDevice
                 sessionContext.device = availableDevice
+                // TODO: AB shouldn't be hardcoded
```

Comment:
TODO? ^_^

Reply:
;D yes, I know. 
But, we've talked about this with Janek, and the question is - can we record sessions with devices other than AB? Should we care?